lib/ui/painting.dart

```diff
@@ -5423,7 +5596,9 @@ class ImmutableBuffer extends NativeFieldWrapperClass1 {
       instance._init(list, callback);
     }).then((_) => instance);
   }
-  void _init(Uint8List list, _Callback<void> callback) native 'ImmutableBuffer_init';
+
+  @FfiNative<Void Function(Handle, Handle, Handle)>('ImmutableBuffer::init')
+  external void _init(Uint8List list, _Callback<void> callback);
```

Comment:
I think that this was a bug - `ImmutableBuffer._init` should have had a `String?` return type.  

To fix this:
* replace this function's return type with `String?`
* change the `instance._init(list, callback)` line above to `return instance._init(list, callback)`
* revert the change to `ImmutableBuffer::init` in `painting/immutable_buffer.cc`.  `ImmutableBuffer::init` should return a string for an error instead of throwing.

Reply:
Good catch. Done!